aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -0,0 +1,170 @@
+## Elements of an App UI
+The User Interface (UI) for an HPE application consists of three 
+building blocks: App Container, Global Header and Footer, and 
```

Comment:
can we list these out to be 
1. App Container
2. Global Header
3. Footer

Reply:
actually just realized they are listed out below so I'm okay with leaving how is, if an example does go here maybe we can link each of these to their explanation below

Reply:
Yeah, we are attempting to gradually unfold the content, starting with the highest level, App, then Page, then Page Section.

The idea is that at each level, there is a single "container" which may have multiple "content" children. The "container's" job is to organize how the children "content" layout, order, respond; the children "content" are only concerned with themselves.

So at the App level, we will have an App Container whose content is Global Header & Footer and a Page Container.

At the Page level, we will have a Page Container whose content is a Page Header and Page Content.

At the Page Header level, it will have a "container" which holds things like the H1, action menu, page level controls, back to parent button, etc.

At the Page Content level, it will have a "container" which controls the ordering of the main "content" sections.